src/components/video/video.js

```diff
@@ -1,45 +1,131 @@
+      loop: undefined // Turn off loop
     };
-    const reducedMotion =
+    this.video = React.createRef();
+  }
+
+  componentDidMount() {
+    // check the users preference
+    const prefersReducedMotion =
       typeof window !== 'undefined'
         ? window.matchMedia('(prefers-reduced-motion: reduce)').matches
         : false;
-    if (reducedMotion) {
-      videoProps = {
-        autoPlay: undefined,
-        loop: undefined,
-        controls: true
-      };
+    // if `prefers-reduced-motion: reduce` is not set, use props to override state defaults
+    if (!prefersReducedMotion) {
+      this.setState({
+        autoPlay: this.props.autoplay,
+        loop: this.props.loop
+      });
     }
+  }
+
+  // show video controls on hover
+  onHover = () => {
+    this.video.current.controls = true;
+  };
+
+  // hide video controls when user leaves the video container
+  onOut = () => {
+    this.video.current.controls = false;
+  };
+
+  // check if video is playing (helps us detect if user has autoplay enabled or disabled)
+  onPlay = () => {
+    this.setState({ isPlaying: true });
+  };
+
+  // check when video stops
+  onStop = () => {
+    this.setState({ isPlaying: false });
+  };
+
+  // play the video
+  playVideo = () => {
+    this.video.current.play();
+  };
+
+  render() {
+    const { src, title, muted, playsInline, poster } = this.props;
+    const { isPlaying, autoPlay, loop } = this.state;
+
+    const videoProps = {
+      autoPlay,
+      loop,
+      muted,
+      playsInline,
+      poster
+    };
+
     return (
-      <div>
+      <div
+        className="relative flex-parent flex-parent--center-main flex-parent--center-cross bg-gray"
+        onMouseOver={this.onHover}
+        onMouseLeave={this.onOut}
+        onFocus={this.onHover}
+      >
+        {!isPlaying && (
+          <button
+            onClick={this.playVideo}
+            className="bg-darken75 bg-darken50-on-hover w60 h60 py12 px12 w120-mm h120-mm py18-mm px18-mm round cursor-pointer z1 absolute"
+          >
+            <VimeoPlayImage />
+          </button>
+        )}
         <video
           {...videoProps}
-          muted
           width="100%"
           className="block mx-auto"
-          src={this.props.src}
+          src={src}
           type="video/mp4"
-          title={this.props.title}
+          title={title}
+          ref={this.video}
+          onPlay={this.onPlay}
+          onEnded={this.onStop}
+          onPause={this.onStop}
         >
           <p>
-            Your browser doesn't support HTML5 video. Here is a{' '}
-            <a href={this.props.src}>link to the video</a> instead.
+            Your browser doesn't support HTML5 video. Open{' '}
+            <a href={this.props.src}>link to the video</a>.
           </p>
         </video>
       </div>
     );
   }
 }
 
+// default props are only set if the user does not prefer reduced motion
+Video.defaultProps = {
+  autoplay: true,
```

Comment:
Should autoplay be `false` by default?

Reply:
@danswick I can help improve the wording, but here's the gist:

1. The `state` by default disables autoplay. We assume that the user will prefer reduced motion.
2. Then in componentDidMount, we check the users's preferences:
    - If the user prefers reduced motion, the component will not use default props and will continue to disable auto play.
    - If the user does not prefer reduced motion, then the component will use default props to determine if the video should be autoplayed.